tests/chart_tests/test_sts_volume_claim_annotations.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+
+from tests import supported_k8s_versions
+from tests.chart_tests.helm_template_generator import render_chart
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+class TestStatefulSetsAnnotations:
+    def test_sts_with_no_annotations(self, kube_version):
+        """Test if the sts contains volume claim templates annotations or not"""
+        sts_workload = [doc for doc in render_chart() if doc["kind"] == "StatefulSet"]
+        assert len(sts_workload) == 7
+
+        with pytest.raises(Exception) as except_info:
+            for doc in sts_workload:
+                if doc["spec"]["volumeClaimTemplates"][0]["metadata"]["annotations"]:
+                    raise Exception("Empty Annotations")
+            assert except_info.value.args[0] == "Empty Annotations"
+            assert str(except_info.value) == "Empty Annotations"
```

Comment:
We can do this without pytest.raises

```suggestion
        vcts = [
            doc["spec"]["volumeClaimTemplates"]
            for doc in render_chart()
            if doc["kind"] == "StatefulSet" and doc["spec"].get("volumeClaimTemplates")
        ]
        assert len(vcts) == 6

        for vct in vcts:
            assert not vct[0]["metadata"].get("annotations")
```